### DISCUSSION TITLE
how to use OrchardCore.Users in an Application without admin theme?

### DISCUSSION THREAD

**Discussion by scil** [OrchardCMS/OrchardCore#11917] [General]
my modules and applications in a sullution are following
```
Me.Application.Admin
Me.Theme.TheAdmin

Me.Application.Reading
Me.Theme.Default

Me.Module.Common
Me.Module.Reading

```

There are two Application modules, `Application.Admin` is with an admin theme `Theme.TheAdmin`, while `Application.Reading` is with a non-admin theme `Me.Theme.Default`.

I have installed `OrchardCore.Users` in the non-admin theme and enalbes features like `"OrchardCore.Users",`   
and got `404` when visiting `/Login` 

what else could I do to visit '/Loign`?

this is my appsettings.json
```
  "OrchardCore": {
    "OrchardCore.Users": {
      "LoginPath": "Login",    
    }
  },

```

**Comment by hishamco** [comment]
There's a login layout that has been used https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheAdmin/Views/Layout-Login.cshtml, you need to override into your theme
